README.md

```diff
@@ -100,7 +100,53 @@ public String TrimString(String s) {
 ## Class members
 
 - Avoid `this.` unless absolutely necessary.
-- Always specify member visiblity, even if it's the default (i.e. `private string foo;` not `string _foo;`).
+
+DO
```

Comment:
Minor: Could we use `&#10003; **DO**` and `&#10007; **DO NOT**` markup for &#10003; **DO** and &#10007; **DO NOT**